src/main/java/com/microsoft/graph/models/AvailabilityItem.java

```diff
@@ -39,13 +40,13 @@ public final AdditionalDataManager additionalDataManager() {
     }
 
     /**
-     * The End Time.
-     * 
+     * The End Date Time.
+     * The end time of the time slot.
      */
-    @SerializedName(value = "endTime", alternate = {"EndTime"})
+    @SerializedName(value = "endDateTime", alternate = {"EndDateTime"})
     @Expose
 	@Nullable
-    public com.microsoft.graph.core.TimeOfDay endTime;
```

Comment:
this is a breaking change. What did you end up doing on dotnet @andrueastman ? CC @darrelmiller .
ICM opened for clarification (still no answer)
https://portal.microsofticm.com/imp/v3/incidents/details/325687689/home

Reply:
We saw this last week on dotnet via https://github.com/microsoftgraph/msgraph-sdk-dotnet/pull/1432#issuecomment-1209703373 and yet to make a new release.

Reply:
By making a new release do you mean bumping major?

Reply:
found the corresponding PR that introduced the change https://dev.azure.com/msazure/One/_git/AD-AggregatorService-Workloads/pullrequest/6315041?_a=files&path=/Workloads/Microsoft.Bookings/schema.csdl

Reply:
Thanks @baywet. From the PR comments it looks like 
- This is a fix for an incorrect type declaration
- The feature is still protected and not yet announced.

This means we can treat this as a fix and go ahead with unblocking the releases without major bump. Any reservations on this?

Reply:
Confirmed, the type was only introduced a week prior by this pr. And the author confirmed it was just a fix. https://dev.azure.com/msazure/One/_git/AD-AggregatorService-Workloads/pullrequest/6288887?path=/Workloads/Microsoft.Bookings/override/schema-Prod-v1.0.csdl
Let's proceed with the change without bumping major